schemas/panels/bar/bar.cue

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bar
+
+import (
+	"github.com/perses/perses/schemas/common"
+)
+
+kind: "BarChart"
+spec: close({
+	calculation: common.#calculation
+	unit?:       common.#unit
+	sort?:       "asc" | "desc"
+	mode?:       "value" | "percentage"
```

Comment:
Do we know roughly how this schema maps to Grafana's [bargauge](https://play.grafana.org/d/ktMs4D6Mk/5-bar-charts-and-pie-charts?orgId=1&editPanel=13&showCategory=Value%20options) supported options?

<img width="760" alt="image" src="https://github.com/perses/perses/assets/9369625/8b4ccda3-2f55-4292-8d79-48d411f2ea23">

Reply:
I don't see a percent mode or anything similar for Grafana's bar gauge. Maybe when we do the conversion we just have to switch it back to the raw values?